src/core/js/models/adaptModel.js

```diff
@@ -524,17 +523,18 @@ export default class AdaptModel extends LockingModel {
 
     for (let i = modelIndex, l = pageDescendants.length; i < l; i++) {
       const descendant = pageDescendants[i];
-      if (descendant.isTypeGroup(relativeDescriptor.type)) {
-        if (movementCount > moveBy) {
-          // there is no descendant which matches this relativeString
-          // probably looking for the descendant 0 in a parent
-          break;
-        }
-        if (movementCount === moveBy) {
-          return Adapt.findById(descendant.get('_id'));
-        }
-        movementCount++;
+      if (!descendant.isTypeGroup(relativeDescriptor.type)) continue;
+      const isSelf = i === modelIndex;
```

Comment:
Can I get some brackets up in here?
```suggestion
      const isSelf = (i === modelIndex);
```